modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/internal/helper/CommerceBasePriceListHelper.java

```diff
@@ -122,7 +122,7 @@ private void _deleteCatalogBaseCommercePriceList(
 					commerceCatalog.getGroupId(), type);
 
 		if (catalogBaseCommercePriceList != null) {
-			_commercePriceListLocalService.deleteCommercePriceList(
+			_commercePriceListLocalService.deleteForceCommercePriceList(
```

Comment:
@riccardo-alberti is this right? @commando18 fyi

Reply:
@andrea-ale-sbarra yes that's the case where we need to force delete the base pricelist

Reply:
thanks